erpnext/accounts/report/statement_of_account/statement_of_account.py

```diff
@@ -25,7 +26,7 @@ def execute(filters=None):
 		account_details.setdefault(acc.name, acc)
 
 	if filters.get('party'):
-		filters.party = frappe.parse_json(filters.get("party"))
+		filters.party = filters.get("party")
```

Comment:
- Its always better to do a `parse_json`

Reply:
I think this is not need we just simply passing one string. also assigning twice is not useful so removing it